src/Application/Actions/Donations/Update.php

```diff
@@ -268,7 +269,7 @@ private function addData(Donation $donation, HttpModels\Donation $donationData,
 
     private function cancel(Donation $donation, Response $response, array $args): Response
     {
-        if ($donation->getDonationStatus() === 'Cancelled') {
+        if ($donation->getDonationStatus() === DonationStatus::Cancelled) {
```

Comment:
We might need to do a global search the entire code base to check all string statuses are replaced

Reply:
Thanks, used regex search for `['"]Reserved['"]`, `['"]Paid['"]` etc. None appear where they shouldn't, they've all been replaced with references to the enum case.